generator/metadata/enum_meta_data.go

```diff
@@ -1,7 +1,20 @@
 package metadata
 
+import "regexp"
+
 // Enum metadata struct
 type Enum struct {
-	Name   string `sql:"primary_key"`
-	Values []string
+	Name    string `sql:"primary_key"`
+	Comment string
+	Values  []string
+}
+
+// GoLangComment returns enum comment without ascii control characters
+func (e Enum) GoLangComment() string {
+	if e.Comment == "" {
+		return ""
+	}
+
+	// remove ascii control characters from string
+	return regexp.MustCompile(`[[:cntrl:]]+`).ReplaceAllString(e.Comment, "")
 }
```

Comment:
This function repeats a lot. Maybe we can covert it to util function `GoLangComment(comment string) -> "// comment"` inside template package, and pass it to template engine using FuncMap, so we can write inside template:
``` 
{{GoLangComment .Comment}} 
type {{structImplName}} struct {
```